test/unit/Fixture/ClassWithMethods.php

```diff
@@ -0,0 +1,17 @@
+<?php
```

Comment:
This file name does not reflect the intended usage - this should be `ClassWithMethodsAndTraitMethods`

Reply:
Done.